sections/dashboard/FuturesMarketsTable/FuturesMarketsTable.tsx

```diff
@@ -27,62 +28,60 @@ import {
 	FuturesMarketAsset,
 } from 'utils/futures';
 
-type FuturesMarketsTableProps = {
-	futuresMarkets: FuturesMarket[];
-};
-
-const FuturesMarketsTable: FC<FuturesMarketsTableProps> = ({
-	futuresMarkets,
-}: FuturesMarketsTableProps) => {
+const FuturesMarketsTable: FC = () => {
 	const { t } = useTranslation();
 	const router = useRouter();
 	const { synthsMap } = Connector.useContainer();
 
-	const synthList = futuresMarkets.map(({ asset }) => asset);
+	const futuresMarkets = useRecoilValue(futuresMarketsState);
+
+	const synthList = futuresMarkets?.map(({ asset }) => asset) ?? [];
```

Comment:
Maybe we can make `futuresMarketState` default to `[]`, so we don't need to do the optional chaining every time we want to use it.